modules/stub/src/stub.js

```diff
@@ -1,211 +1,286 @@
-(function() {
+((win, doc) => {
```

Comment:
@chrispaterson will this file be removed and only `StubCmpApi.ts` will be taken into account?